common/data_refinery_common/models/organism.py

```diff
@@ -187,5 +191,20 @@ def get_objects_with_qn_targets(cls):
         """ Return a list of Organisms who already have valid QN targets associated with them. """
         return Organism.objects.all().filter(qn_target__isnull=False)
 
+    def get_most_recent_qn_target(self):
```

Comment:
Do we need this method? I think we save the most recent QN Target in `organism.qn_target`

Reply:
Good point.... that solves the issue I was having with import rings.

Good catch, wish we had realized that earlier lol